test/helpers/types.ts

```diff
@@ -6,11 +6,57 @@ import {
   ZNSPriceOracle,
   ZNSRegistry,
   ZNSTreasury,
-  ZeroTokenMock, ZNSAccessController,
+  ZeroTokenMock,
+  ZNSAccessController,
+  ZNSEthRegistrarMock,
+  ZNSPriceOracleMock,
+  ZNSAddressResolverMock,
+  ZNSDomainTokenMock,
+  ZNSRegistryMock,
+  ZNSTreasuryMock,
+  ZNSAddressResolverMock__factory,
+  ZNSDomainTokenMock__factory,
+  ZNSEthRegistrarMock__factory,
+  ZNSPriceOracleMock__factory,
+  ZNSRegistryMock__factory,
+  ZNSTreasuryMock__factory,
 } from "../../typechain";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
 
 export type Maybe<T> = T | undefined;
 
+export type GetterFunction =
+  (
+    Promise<string> |
+    Promise<boolean> |
+    Promise<BigNumber> |
+    Promise<Array<BigNumber>>
```

Comment:
would this be better if the type parameter is a union type rather than the parent type?

`Promise<string | boolean | ...>` ??